docs/src/pages/component-status.mdx

```diff
@@ -14,7 +14,7 @@ export default ({ children }) => {
   return (
     <Layout
       description="A detailed breakdown of our components and status of their implementation."
-      title="Component status 🚀"
+      title="Component status 🚀 | Orbit"
```

Comment:
If the site title is in Layout, this change is unnecessary. This seems like a less programmatic solution than adding in an SEO title field.